### ISSUE TITLE
[HOWTO] How to use different database/table on the same instance

### ISSUE THREAD

**Issue by borzaka** [rogeraabbccdd/Kento-Rankme#124]
I would like to have different databases/tables for these two game modes, but on the same CS:GO instance:
- DM
- DM pistol only

How can it be achieved with SQLite? I saw the `rankme_sql_table` cvar, but I'am afraid that's only for MySQL.

Or it could work?
- Instance 1.: `rankme_sql_table "rankmedm"`
- Instance 2.: `rankme_sql_table "rankmedmpistol"`

Edit: I have tried, but on the 2nd instance it stayed the default I set in `cfg/sourcemod/kento.rankme.cfg`
I have added this `rankme_sql_table "rankmedmpistol"` to the 2nd instances `server.cfg`.

If I execute this command in the console after the server is up and running on the 2nd instance, it switches database correctly, but not from the `server.cfg` at startup.
How could I use this other `rankme_sql_table` cvar on the 2nd instance?

I was able to successfully change other plugin (DM) settings like `dm_gun_menu_mode`, `dm_armor` and `dm_armor_full` in the 2nd instances `server.cfg`, but not the Rankme's `rankme_sql_table` cvar.

**Comment by rogeraabbccdd** [comment]
Hi, you can only use 1 database at the same time.
If you want to use different databases for different game modes, you can use [map prefix plugin](https://forums.alliedmods.net/showthread.php?t=69506) to load your `rankme_sql_table` by map name.
Or if you are using [UMC](https://forums.alliedmods.net/showthread.php?t=134190), you can add commands for map groups.

**Comment by borzaka** [comment]
I don't think you fully understand me.
Yes, I have different config files for the different servers, but they share the same game files and configs, so they are instances.
And I set different `rankme_sql_table` for the different game modes, but it's not working, only manually entering to the console when the map is loaded.

I ended up writing a plugin with a 5 seconds timer to change that `rankme_sql_table` to the desired value. And it's working.
I think this should work without a custom plugin and only setting `rankme_sql_table` in the configs should work as well.
Maybe some race conditions, or something else.
